UnityProject/Assets/Scripts/Objects/Doors/FalseWallTileBlend.cs

```diff
@@ -47,14 +49,14 @@ private void Start()
 			UpdateWallSprite();
 			ChangeNearFalseWallSprites(GetNearFalseWalls(falseWallPosition));
 
-			UpdateManager.Add(CallbackType.LATE_UPDATE, UpdateMethod);
+			tileChangeManager.MetaTileMap.Layers[LayerType.Walls].onTileMapChanges.AddListener(UpdateMethod);
 		}
 
 		private void OnDisable()
 		{
 			UpdateManager.Remove(CallbackType.LATE_UPDATE, UpdateMethod);
 			falseWallPresented.Remove(falseWallPosition);
-			tileChangeManager.MetaTileMap.RemoveTileWithlayer(falseWallPosition, LayerType.Walls);
+			tileChangeManager.MetaTileMap.Layers[LayerType.Walls].RemoveTile(falseWallPosition);
```

Comment:
I think it's not networked if you remove the tile this way

Reply:
Hmmm, maybe but in test all was right

Reply:
was it between build and editor?

Reply:
Yep